libs/content/workspace/src/tab/markdown_editor/widget/mod.rs

```diff
@@ -11,7 +11,7 @@ pub(crate) mod inline;
 pub(crate) mod toolbar;
 pub(crate) mod utils;
 
-pub const MARGIN: f32 = 45.0; // space between the editor and window border; must be large enough to accommodate bordered elements e.g. code blocks
+pub const MARGIN: f32 = 35.0; // space between the editor and window border; must be large enough to accommodate bordered elements e.g. code blocks
```

Comment:
Does this affect top and bottom margin? If so - we want to adjust the swift constants accordingly (or maybe we should be deriving them from rust directly now?)

Reply:
Good catch!